easybuild/easyconfigs/s/Score-P/Score-P-5.0-gompi-2019a.eb

```diff
@@ -0,0 +1,62 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# Copyright:: Copyright 2013-2019 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Score_minus_P'
```

Comment:
You can actually remove this, since it matches the software name?

Reply:
Wasn't there a policy change at some point in the past to make the easyblock explicit?

Reply:
A (long) time, we dropped the automagic fallback to trying with `ConfigureMake` (which was confusing since that always left you wondering which easyblock was actually being used if none was specified).

But as long as the software name matches the easyblock, there's no point in specifying it, since there's no ambiguity.